Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -14,7 +14,10 @@ public protocol CommentRepliesViewModelInputs {
    **/
   func configureWith(comment: Comment, project: Project, inputAreaBecomeFirstResponder: Bool)
 
-  /// Call when the view appears.
+  /// Call when a `ViewMoreRepliesCell` on the `CommentRepliesViewController` is selected.
+  func viewMoreRepliesCellWasTapped()
```

Comment:
Tiny thing, I don't think we need to specify that it's a `Cell`. Just because it's more like a user action, `viewMoreRepliesTapped`.

Reply:
Going to leave it as is because this input is called, if and only if, the `ViewMoreRepliesCell` is selected.